internal/destination/writer.go

```diff
@@ -90,3 +90,12 @@ func (w *Writer) Close() error {
 
 	return nil
 }
+
+// Reconnect rebuilds jetstream context
+func (w *Writer) Reconnect(conn *nats.Conn) (err error) {
```

Comment:
Where is this method actually being used?

Reply:
It is not anymore, I will remove it.